packages/web/src/common/store/cache/tracks/utils/retrieveTracks.ts

```diff
@@ -77,19 +85,13 @@ export function* retrieveTrackByHandleAndSlug({
     },
     retrieveFromSource: function* (permalinks: string[]) {
       yield* waitForRead()
-      const apiClient = yield* getContext('apiClient')
+      const sdk = yield* getSDK()
       const userId = yield* select(getUserId)
-      const track = yield* call((args) => {
-        const split = args[0].split('/')
```

Comment:
Interesting that we were doing all this just to call the wrapper around `getBulkTracks` 🤔

Reply:
There are a number of places where we are calling legacy endpoints with handle/slug and other variations of parameters that are now deprecated.